src/lesson6/Circle.java

```diff
@@ -0,0 +1,13 @@
+package lesson6;
+
+public class Circle {
+    private final int RADIUS;
+
```

Comment:
у великому регістрі тільки константи
```suggestion
    private final int radius;

```